www/src/pages/foundations/spacing.tsx

```diff
@@ -127,7 +133,7 @@ export default function SpacingPage() {
                   <Input
```

Comment:
[curious] Since the `Input` component is [deprecated](https://paragon-openedx.netlify.app/components/input/), should we migrate from `Input` to `Form.Control` or otherwise here?

Reply:
Done